app/models/loan.rb

```diff
@@ -0,0 +1,4 @@
+class Loan < ActiveRecord::Base
+	belongs_to :asset
+	enum status_types: [:Loaned, :Overdue, :Returned]
```

Comment:
Was using status_types as status is being used for a vareible to capture data to the database